hasura/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM hasura/graphql-engine:v1.2.1.cli-migrations-v2
+ENV HASURA_GRAPHQL_ENABLE_TELEMETRY false
+COPY ./migrations /hasura-migrations
```

Comment:
```suggestion
COPY ./migrations /hasura-migrations
COPY ./metadata /hasura-metadata
```

Reply:
I don't understand why metadata are not needed ?

Reply:
its been commited